tools/model_tools/src/openvino/model_zoo/download_engine/cache.py

```diff
@@ -39,17 +41,17 @@ def __init__(self, cache_dir):
         self._staging_dir = self._cache_dir / 'staging'
         self._staging_dir.mkdir(exist_ok=True)
 
-    def _hash_path(self, hash):
-        assert len(hash) == self._HASH_LEN
+    def _hash_path(self, hash, hash_len):
+        assert len(hash) == hash_len
```

Comment:
That's strange, just for a debug assertion you've changed the interface of three functions. May be remove the assertion?

Reply:
Done